src/java.base/share/classes/sun/net/InetAddressCachePolicy.java

```diff
@@ -149,6 +159,12 @@ public Integer run() {
             negativeCachePolicy = tmp < 0 ? FOREVER : tmp;
             propertyNegativeSet = true;
         }
+/*[IF CRIU_SUPPORT]*/
+        if (InternalCRIUSupport.isCRIUSupportEnabled()) {
+            InternalCRIUSupport.initInetAddressCachePolicy = cachePolicy;
+            cachePolicy = NEVER;
```

Comment:
if you do this in the static initializer, doesn't it make it permanent?

Reply:
The `cachePolicy` is set to `NEVER` only when `-XX:+EnableCRIUSupport` is specified, i.e., if the JVM prepares to do a checkpoint, no cache will be saved.
After the restoring, a post-restore hook implementated in following OpenJ9 PR is going to set `cachePolicy` to the initial value saved.
* https://github.com/eclipse-openj9/openj9/pull/17448

Reply:
Updated along with 
* https://github.com/eclipse-openj9/openj9/pull/17448